__tests__/functional/tagged-format.test.ts

```diff
@@ -99,17 +99,17 @@ describe("tagged format", () => {
 
   it("handleds conflicts", () => {
     var result = TaggedTypeFormat.encode({
-      "@date": new Date(2022, 11, 1),
-      "@time": new Date(2022, 11, 2),
-      "@int": 1,
-      "@long": 9999999999999,
-      "@double": 1.99,
+      date: { "@date": new Date(2022, 11, 1) },
```

Comment:
here, `"@date"` is supposed to be a user-field, right? But then we assign to it a `Date`. Should the `Date` still get encoded as `{ "@date": "2022-11-01" }`?

I think the test should then be

```
expect(result["date"]["@object"]["@date"]).toEqual({"@date": "2022-11-01"});
```